AppBuilder/platform/views/ABViewDetailText.js

```diff
@@ -110,6 +110,15 @@ module.exports = class ABViewDetailText extends ABViewDetailTextCore {
 
       if (this.settings.height) component.ui.height = this.settings.height;
 
+      component.ui.on = {
+         //Add data-cy attribute for Cypress Testing
+         onAfterRender: () => {
+            const dataCy = `detailText-${field?.label?.replace(/\s/g,'')}-${field?.id}`;
+            document.querySelector(`[view_id="${ids.component}"]`)
```

Comment:
Let's try to use `$$(ids.component).$view` or remove the` =>` and use `function() {` on line 116 and try `this.$view`. If either works it may be a good way to do it on the others above as well.